pf/tests/testdata/basicprogram/Pulumi.yaml

```diff
@@ -196,3 +203,5 @@ outputs:
       "Hello"
     sensitive:
       "Hello"
+
+  testSecret__secret: ${secretRes.requiredInputString}
```

Comment:
I'd actually prefer to see some redundancy here instead of extending the framework, but perhaps this doesn't work?

```
  testSecret__actual: ${secretRes.requiredInputString}
  testSecret__expect:
      fn::secret: "secretInput"
```

Reply:
```
                                expected: map[string]interface {}{"4dabf18193072939515e22adb298388d":"1b47061264138c4ac30d75fd1eb44270", "ciphertext":"v1:M0qQ+UTHuIr3vZAC:6LQecGeqkvHGvumKCRPkeGdwsaCBKJ5jC/iTLVI="}
                                actual  : map[string]interface {}{"4dabf18193072939515e22adb298388d":"1b47061264138c4ac30d75fd1eb44270", "ciphertext":"v1:TkrbDFC8wSV02NBu:b36ZaxdBFrui4XWEZbsznokoqLs1LXg1j5zWC/E="}

```

Ar whoops it doesn't work. What happened here, different encrypt salt?

Reply:
I see why you need to customize the framework.

Reply:
Yeah, I tried that. I also tried inlining the secret map, but that resulted in a serialization error.